src/main/java/net/snowflake/ingest/streaming/internal/ColumnMetadata.java

```diff
@@ -111,6 +111,10 @@ boolean getNullable() {
     return this.nullable;
   }
 
+  String getInternalName() {
```

Comment:
Should we just store the internal name as well? Looks like there're a bunch of places are using it

Reply:
Good point, I changed it to calculate and store the internal name from name when `setName` is called. In the future, we will read it from the server.